Gemfile

```diff
@@ -1,8 +1,12 @@
+# Remplazed frozen_string_literal true for something else
+
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.2.2'
 
+# gem 'sqlite3'
+
```

Comment:
![image](https://github.com/EmiLoKiJu/BlogApp/assets/93039359/95a2cc85-6dde-4207-872c-b7f5e8f21e9a)

- [ ] Kindly put the necessary commands to the highlighted sections in the image above to make your readme descriptive and useful 👍